docs/features/multiple-networks/multi-homing.md

```diff
@@ -138,7 +207,7 @@ spec:
   config: |2
     {
             "cniVersion": "1.0.0",
-            "name": "localnet-network",
+            "name": "localnet-network-overlay",
```

Comment:
wrong. 

it's not an overlay. it uses the physical network.

Reply:
it's the name of the ovn-k CNI binary in the hosts.

You're right it may no longer fit all the use cases, but that's besides the point.